src/asciidoc/docs/refs/glossary.adoc

```diff
@@ -0,0 +1,33 @@
+= Glossary
+:page-slug: glossary
+:page-description: Definition of terms
+// Settings:
+:experimental:
+// Refs:
+:includespath: ../_includes
+
+entitlement::
+Configuration settings that allow the template or Koji to use certain platform features, such as enabling remixing or defining product information for in-app purchases.
+
+in-app purchase (IAP)::
+A financial transaction made within a Koji.
+
+Koji::
+A mini web application that can be shared on all social platforms, embedded in any website, and sent via messengers and email.
+Internally, a Koji is comprised of the associated template and the custom remix settings.
+Learn more: <<introduction#>>
+
+player chrome::
+The Koji button on a user's window.
+
+remix::
+The process of customizing a Koji or template.
+Or the custom Koji resulting from a remix process.
+Learn more: <<instant-remixing#>>
+
+template::
+The underlying web application that powers a Koji and its remixes.
```

Comment:
This definition is probably outdated. We'll need to review it with Sean and make some decisions about how we want to update the docs moving forward.

Reply:
Removed.